apps/src/applab/commands.js

```diff
@@ -52,6 +52,23 @@ function outputError(errorString) {
   errorHandler.outputError(errorString, ErrorLevel.ERROR, line);
 }
 
+/**
+ * Returns the error handler specified by the user if one exists. Otherwise
+ * returns an error handler which prints warnings to the applab console,
+ * with line numbers which are accurate even for async callbacks.
+ * @param {function{string)} onError The error handler supplied by the user.
+ * @returns {function(*)}
+ */
+function getErrorHandler(onError) {
```

Comment:
Seems strange to pass onError in here, only to return it if we do so. Would it make more sense to have callers do
```
var onError = opts.onError || getErrorHandler();
```